lib/route_translator/translator/path/segment.rb

```diff
@@ -8,14 +8,10 @@ def translate_string(str, locale, scope)
             opts = { scope: scope, locale: locale }
             res = I18n.translate(str, opts)
```

Comment:
I don't like this, we need to find another way.

Line 9 and line 15 are doing the same thing, we should check and set options first and then assign `res`